CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 #### Upcoming Changes
 
+* Add missing hint on cairo_secp lib [#984]:
+    `BuiltinHintProcessor` now supports the following hint:
```

Comment:
Please add an enter between these two lines so they don't appear in the same line